examples/api/lib/widgets/sliver/decorated_sliver.1.dart

```diff
@@ -0,0 +1,158 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+/// Flutter code example for [DecoratedSliver]
+/// with clipping turned off in a parent [CustomScrollView].
+
+void main() => runApp(const DecoratedSliverClipExampleApp());
+
+class DecoratedSliverClipExampleApp extends StatelessWidget {
+  const DecoratedSliverClipExampleApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'DecoratedSliver Clip Example',
+      theme: ThemeData(
+        colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple),
+        useMaterial3: true,
+      ),
+      home: const DecoratedSliverClipExample(),
+    );
+  }
+}
+
+class DecoratedSliverClipExample extends StatefulWidget {
+  const DecoratedSliverClipExample({super.key});
+
+  @override
+  State<DecoratedSliverClipExample> createState() => _DecoratedSliverClipExampleState();
+}
+
+class _DecoratedSliverClipExampleState extends State<DecoratedSliverClipExample> {
+  double _height = 225.0;
+  bool isClipped = false;
```

Comment:
super tiny nits. lets keep the styling consistent
```suggestion
  bool _isClipped = false;
```